CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,85 @@
+  * Use the `transportInterfaceCIDRs` configuration parameter for the Antrea Agent to choose an interface by network CIDRs
+- Add UDP support for NodePortLocal. ([#2448](https://github.com/antrea-io/antrea/pull/2448), [@chauhanshubham])
+- Add more visibility metrics about connection status for Flow Aggregator. ([#2668](https://github.com/antrea-io/antrea/pull/2668), [@zyiou])
+- Add the `antreaProxy.skipServices` configuration parameter for the Antrea Agent to specify Services which should be ignored by AntreaProxy. ([#2882](https://github.com/antrea-io/antrea/pull/2882), [@luolanzone])
+  * A typical use case is setting `antreaProxy.skipServices` to `["kube-system/kube-dns"]` to make [NodeLocal DNSCache] work when AntreaProxy is enabled.
+- Add support for `ToServices` in the rules of Antrea-native policies to allow matching traffic intended for Services. ([#2755](https://github.com/antrea-io/antrea/pull/2755), [@GraysonWu])
+- Add the `egress.exceptCIDRs` configuration parameter for the Antrea Agent to specify traffic to which SNAT should not be performed. ([#2749](https://github.com/antrea-io/antrea/pull/2749), [@leonstack])
+- Add user documentation for [WireGuard encryption]. ([#2902](https://github.com/antrea-io/antrea/pull/2902), [@jianjuns])
+- Add user documentation for [encap mode installation for EKS]. ([#2929](https://github.com/antrea-io/antrea/pull/2929), [@jianjuns])
+- Add the `nodePortLocal.enable` configuration parameter fot the Antrea Agent to enable NodePortLocal. ([#2924](https://github.com/antrea-io/antrea/pull/2924), [@antoninbas])
+
+### Changed
+
+- Remove chmod for OVSDB file from start_ovs, as the permissions are set correctly by OVS 2.15.1. ([#2803](https://github.com/antrea-io/antrea/pull/2803), [@antoninbas])
+- Reduce memory usage of antctl when collecting supportbundle. ([#2813](https://github.com/antrea-io/antrea/pull/2813), [@tnqn])
+- Do not perform SNAT for egress traffic to Kubernetes Node IPs. ([#2762](https://github.com/antrea-io/antrea/pull/2762), [@leonstack])
+- Change flow exporter export expiry mechanism to priority queue based, to reduce CPU usage and memory footprint. ([#2360](https://github.com/antrea-io/antrea/pull/2360), [@heanlan])
+- Send gratuitous ARP for EgressIP via the transport interface, as opposed to the interface with Node IP (if they are different). ([#2845](https://github.com/antrea-io/antrea/pull/2845), [@Jexf])
+- Wait for AntreaProxy to be ready before accessing Service if `antreaProxy.proxyAll` is enabled, to avoid connection issues on Agent startup. ([#2858](https://github.com/antrea-io/antrea/pull/2858), [@tnqn])
+- Ignore hostNetwork Pods selected by Egress, as they are not supported. ([#2851](https://github.com/antrea-io/antrea/pull/2851), [@Jexf])
+- Add windows task delay up to 30s to improve job resiliency (documentation) ([#2864](https://github.com/antrea-io/antrea/pull/2864), [@perithompson]) [Windows]
```

Comment:
windows -> Windows

Reply:
I feel this one is too small to mention. If we want to keep it, probably explain what tasks/jobs.